jib-gradle-plugin/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fixes issue with `jibBuildTar` where `UP-TO-DATE` checks were incorrect ([#1757](https://github.com/GoogleContainerTools/jib/issues/1757))
```

Comment:
Let's use past tense "Fixed" to be consistent with the rest of the changelog

Reply:
:+1: